scripts/apply-track-config.jl

```diff
@@ -3,6 +3,32 @@ using TOML
 # Parse config files
 global_cfg = TOML.parsefile(joinpath("main", "config.toml"))
 
+# Track config
+# Check for existence of track config file
+track_cfg_file = joinpath("track-repo", ".github", "org-wide-files-config.toml")
+if isfile(track_cfg_file)
+    track_cfg = TOML.parsefile(track_cfg_file)
+
+    # Configlet
+    if haskey(track_cfg, "configlet")
+        # Iterate through allowed configlet settings
+        for key in track_cfg["configlet_configurable_keys"]
+            haskey(track_cfg["configlet"], key) || continue
+
+            if track_cfg["configlet"]["fmt"]
+                @info "Track wants to run configlet fmt; appending configlet workflow"
+
+                open(joinpath("track-repo", ".github", "workflows", "configlet.yml"), "a") do io
+                    write(io, "    with:\n      fmt: true\n")
```

Comment:
I wonder if the workflow should have

```yaml
with:
  # This can be configured via .github/org-wide-files-config.toml
  # See exercism/github-actions README for more information
  fmt: false
```

by default so that people are made aware of this option.

Reply:
At some point this would be nice. For now, I'd prefer not to as configlet fmt is only formatting exercise configs and is missing concept and config.json formatting capabilities.